### DISCUSSION TITLE
186

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#157] [General]
# 潮流周刊第186期 - 青柠冷饮

封面图拍摄于早上做的的口粮，很好喝，3 块冰块，3 颗小青柠剪开，半杯苏打水半杯纯椰子水，可以试试，好看还好喝。

https://weekly.tw93.fun/posts/186-%E9%9D%92%E6%9F%A0%E5%86%B7%E9%A5%AE

**Comment by Vainycos** [comment]
周一打卡

**Comment by josangmi** [comment]
不能上图片的么。 做的大字报上不来了

**Reply by tw93** [reply]
可以的 你试试 cdn
